### ISSUE TITLE
FYI: Jekyll custom plugins that are banned by github have implementation in standard library

### ISSUE THREAD

**Issue by akuleshov7** [yegor256/blog#465]
This is just FYI, so you (or someone else like @olegmoz or @g4s8) will be able to workaround it in case of any problems.

For example: `reading_time.rb` will not work with github as it is not white listed. 
So all calls from `index.html` will not be executed:
```
<li itemprop="wordCount">
              {{ post.content | count_words }} words
            </li>
            <li>
              {% capture time %}{{ post.content | reading_time }}{% endcapture %}
              {% if time == '1' %}
                a minute
              {% else %}
                {% case time %}
                  {% when '2' %}
                    two
                  {% when '3' %}
                    three
                  {% when '4' %}
                    four
                  {% when '5' %}
                    five
                  {% else %}
                    {{ time }}
                {% endcase %}
                minutes
              {% endif %}
              to read
            </li>
```

In this case you can **workaround** it with function from the standard library and Liquid:
```
<!-- counting the number of words -->
        <li>
            {{ content | number_of_words | append: ' words' }}
        </li>
```

```
<!-- counting the reading time -->
        <li>
            {% capture words %}
            {{ content | number_of_words | minus: 180 }}
            {% endcapture %}
            {% unless words contains '-' %}
            {{ words | plus: 180 | divided_by: 180 | append: ' minutes to read' }}
            {% endunless %}
        </li>
```

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@58f448d9-9e86-4732-856c-326f8d64455a, thread: pool-961-thread-1
org.hamcrest.MatcherAssert[20] java.lang.AssertionError: HTTP response status is not equal to 201:
404 Not Found [https://api.github.com/repos/yegor256/blog/labels]
Server: GitHub.com
Date: Thu, 12 Nov 2020 17:25:32 GMT
Content-Type: application/json; charset=utf-8
Transfer-Encoding: chunked
Status: 404 Not Found
X-GitHub-Media-Type: github.v3
X-RateLimit-Limit: 5000
X-RateLimit-Remaining: 4913
X-RateLimit-Reset: 1605203494
X-RateLimit-Used: 87
Access-Control-Expose-Headers: ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Used, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, Deprecation, Sunset
Access-Control-Allow-Origin: *
Strict-Transport-Security: max-age=31536000; includeSubdomains; preload
X-Frame-Options: deny
X-Content-Type-Options: nosniff
X-XSS-Protection: 1; mode=block
Referrer-Policy: origin-when-cross-origin, strict-origin-when-cross-origin
Content-Security-Policy: default-src 'none'
Vary: Accept-Encoding, Accept, X-Requested-With
X-GitHub-Request-Id: BE46:6783:67BC41:105311B:5FAD700C

{"message":"Not Found","documentation_url":"https://docs.github.com/rest/reference/issues#create-a-label"}
Expected: HTTP response with status 201
     but: was <404 Not Found [https://api.github.com/repos/yegor256/blog/labels]
Server: GitHub.com
Date: Thu, 12 Nov 2020 17:25:32 GMT
Content-Type: application/json; charset=utf-8
Transfer-Encoding: chunked
Status: 404 Not Found
X-GitHub-Media-Type: github.v3
X-RateLimit-Limit: 5000
X-RateLimit-Remaining: 4913
X-RateLimit-Reset: 1605203494
X-RateLimit-Used: 87
Access-Control-Expose-Headers: ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Used, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, Deprecation, Sunset
Access-Control-Allow-Origin: *
Strict-Transport-Security: max-age=31536000; includeSubdomains; preload
X-Frame-Options: deny
X-Content-Type-Options: nosniff
X-XSS-Protection: 1; mode=block
Referrer-Policy: origin-when-cross-origin, strict-origin-when-cross-origin
Content-Security-Policy: default-src 'none'
Vary: Accept-Encoding, Accept, X-Requested-With
X-GitHub-Request-Id: BE46:6783:67BC41:105311B:5FAD700C

{"message":"Not Found","documentation_url":"https://docs.github.com/rest/reference/issues#create-a-label"}>
```

[0.54.5](https://github.com/zerocracy/farm): Issue: yegor256/blog#465, Action: `opened`

**Comment by yegor256** [comment]
@akuleshov7 thanks, fixed in 109f73a5bf0
